fmts/doc.go

```diff
@@ -1,7 +1,9 @@
 // Package fmts holds defined errorformats.
 //
 // Defined formats:
-// 
+//
+// 	css
+// 		stylelint	A mighty modern CSS linter - https://github.com/stylelint/stylelint
```

Comment:
Could you use `go generate ./...` to generate document. 
https://github.com/haya14busa/errorformat/tree/master/fmts#4-go-generate-